plugins/techdocs/src/reader/transformers/addBaseUrl.test.ts

```diff
@@ -125,4 +128,60 @@ describe('addBaseUrl', () => {
       });
     });
   });
+
+  it('inlines absolute url svgs pointed at our backend', async () => {
+    const svgContent = '<svg></svg>';
+    const expectedSrc = `data:image/svg+xml;base64,${Buffer.from(
+      svgContent,
+    ).toString('base64')}`;
+
+    (global.fetch as jest.Mock).mockReturnValue({
+      text: jest.fn().mockResolvedValue(svgContent),
+    });
+
+    const root = createTestShadowDom(
+      `<img id="x" src="${API_ORIGIN_URL}/test.svg" />`,
+      {
+        preTransformers: [
+          addBaseUrl({
+            techdocsStorageApi,
+            entityId: mockEntityId,
+            path: '',
+          }),
+        ],
+        postTransformers: [],
+      },
+    );
+
+    await new Promise<void>(done => {
+      process.nextTick(() => {
```

Comment:
wait shouldn't this just use `await waitFor(() =>` from react-testing-library

Reply:
+1, or another option if you just need to wait a tick is to `await Promise.resolve()`

Reply:
Makes sense for this + the case above.  Interestingly, turns out it's a bad fit for the case below though, where my initial state and my expectations are identical (because what I really want to test is that, after code is run, nothing changes).

- Because `waitFor()` fires immediately, it passes on old code.
- And a plain old `await Promise.resolve()` also doesn't do the trick, probably because the underlying code is `await`s within a `forEach`.  (Though it works with 3 instances of `await Promise.resolve()` prepended to the assertions).